keyboards/zeal60/rgb_backlight.c

```diff
@@ -1019,15 +1019,15 @@ void backlight_effect_all_off(void)
 void backlight_effect_solid_color(void)
 {
     HSV hsv = { .h = g_config.color_1.h, .s = g_config.color_1.s, .v = g_config.brightness };
-    RGB rgb = hsv_to_rgb( hsv );
```

Comment:
These should stay as RGB not LED_TYPE... they are never anything other than RGB.

Reply:
The hsv_to_rgb function now returns LED_TYPE, and in the zeal60 case it would return RGB as RGBW is not define. So these could be left as RGB and just error if RGBW was defined, though I'd rather move to a unified API and keep it as LED_TYPE, though since the Zeal60 is your board I'll lean towards your preference on this.

Reply:
reverted